apps/admin-portal/src/pages/identity-provider-edit.tsx

```diff
@@ -27,7 +27,7 @@ import { IdentityProviderConstants } from "../constants";
 import { useDispatch } from "react-redux";
 import { addAlert } from "@wso2is/core/store";
 import { AlertLevels } from "@wso2is/core/models";
-import { EditIdentityProvider } from "../components/identityProviders/identity-provider-edit";
+import { EditIdentityProvider } from "../components/identity-providers/identity-provider-edit";
```

Comment:
Can we export this component from `components` index. Then we'll be able to shorten the import as follows.

```suggestion
import { EditIdentityProvider } from "../components";
```

Reply:
Fixed with a71ec75bd7ff9034c30a22f8fd552a5d61bc892e.